src/main/pages/che-7/end-user-guide/proc_writing-a-devfile-for-your-project.adoc

```diff
@@ -1,27 +1,81 @@
 [id="writing-a-devfile-for-your-project_{context}"]
 = Writing a Devfile for your project
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section describes how to create a minimal devfile for your project and how to include more than one projects in a devfile.
 
-[discrete]
-.Prerequisites
+// .Prerequisites
+// 
+// * A bulleted list of conditions that must be satisfied before the user starts following this assembly.
+// * You can also link to other modules or assemblies the user must follow before starting this assembly.
+// * Delete the section title and bullets if the assembly has no prerequisites.
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
 
-[discrete]
-.Procedure
+[id="preparing-a-minimal-devfile_{context}"]
+== Preparing a minimal devfile
 
-. Start each step with an active verb.
+A minimal devfile sufficient to run a workspace consists of the following parts:
 
-. Include one command or action per step.
+* Specification version
+* Name
+
```

Comment:
Should we put an example (a reference to example) here?

Reply:
+1 to @skabashnyuk comment. Something as basic as

```yaml
apiVersion: 1.0.0
metadata:
  name: minimal-workspace
```

Reply:
Well, there the petclinic example shortly after. But yes, a minimal example would be nice, too.

Reply:
Specification version is now called "API version" (or at least the element is called `apiVersion`) and name is nested inside a `metadata` element.

So I would reformulate this whole section to something like:

A minimal devfile sufficient to run a workspace consists of nothing more but its name and API version as in the following example:

```yaml
apiVersion: 1.0.0
metadata:
  name: MyDevfile
```

Reply:
added Mario's suggestion